### DISCUSSION TITLE
想了解一下内存占用问题

### DISCUSSION THREAD

**Discussion by pmkol** [IrineSistiana/mosdns#403] [Q&A]
每1万条记录，对应占用内存多少，我记得作者好像之前在哪讲过，找不到那个提问了，重新开了个

另外想了解下，加载插件过多，但如果插件只是作为备用项在配置文件中加载，实际并未使用，是否会增加内存占用，或者对DNS程序的性能造成影响？

例如，我使用fast_forward开了多组TAG，但实际上在sequence中我只用到了某一个TAG，这种情况下会占用内存或影响性能么？

**Comment by IrineSistiana** [comment]
没影响。

每1万条记录，对应占用内存多少

每个平台不一样。载入到列表不一样大小也不一样。自己试试就知道了。
